packages/augur-ui/src/modules/markets/actions/load-markets.ts

```diff
@@ -128,12 +128,16 @@ export const loadMarketsByFilter = (
     universe: universe.id,
     categories: filterOptions.categories,
     search: filterOptions.search ? filterOptions.search : '',
-    maxFee: filterOptions.maxFee === MAX_FEE_100_PERCENT ? '' : filterOptions.maxFee,
+    maxFee:
+      filterOptions.maxFee === MAX_FEE_100_PERCENT ? '' : filterOptions.maxFee,
     includeInvalidMarkets: filterOptions.includeInvalidMarkets,
     limit: filterOptions.limit,
     offset: paginationOffset * filterOptions.limit,
     reportingStates,
-    maxLiquiditySpread: filterOptions.maxLiquiditySpread as Getters.Markets.MaxLiquiditySpread,
+    maxLiquiditySpread:
+      filterOptions.maxLiquiditySpread === MAX_SPREAD_ALL_SPREADS
+        ? ''
```

Comment:
I don't think you can pass an empty string as a value, according to FeaturePeek.
`maxLiquiditySpread: "0" | "10" | "15" | "20" | "100"`

```javascript
ww-connector.ts:123 Uncaught (in promise) Error: Invalid request object: Invalid value "" supplied to : (({ universe: string } & Partial<{ creator: string, search: string, reportingStates: Array<string>, designatedReporter: string, maxFee: string, maxEndTime: number, maxLiquiditySpread: "0" | "10" | "15" | "20" | "100", includeInvalidMarkets: boolean, categories: Array<string>, sortBy: "marketOI" | "liquidity" | "volume" | "timestamp" | "endTime" | "lastTradedTimestamp" | "disputeRound" | "totalRepStakedInMarket", userPortfolioAddress: string }>) & Partial<{ sortBy: string, isSortDescending: boolean, limit: number, offset: number }>)/0: ({ universe: string } & Partial<{ creator: string, search: string, reportingStates: Array<string>, designatedReporter: string, maxFee: string, maxEndTime: number, maxLiquiditySpread: "0" | "10" | "15" | "20" | "100", includeInvalidMarkets: boolean, categories: Array<string>, sortBy: "marketOI" | "liquidity" | "volume" | "timestamp" | "endTime" | "lastTradedTimestamp" | "disputeRound" | "totalRepStakedInMarket", userPortfolioAddress: string }>)/1: Partial<{ creator: string, search: string, reportingStates: Array<string>, designatedReporter: string, maxFee: string, maxEndTime: number, maxLiquiditySpread: "0" | "10" | "15" | "20" | "100", includeInvalidMarkets: boolean, categories: Array<string>, sortBy: "marketOI" | "liquidity" | "volume" | "timestamp" | "endTime" | "lastTradedTimestamp" | "disputeRound" | "totalRepStakedInMarket", userPortfolioAddress: string }>/maxLiquiditySpread: "0" | "10" | "15" | "20" | "100"
    at eval (ww-connector.ts:123)
    at Array.forEach (<anonymous>)
    at Worker.worker.onmessage (ww-connector.ts:121)
```

Reply:
I can delete the property instead of passing empty

Reply:
Didn't work for maxLiquiditySpread, at least on FeaturePeek

Isn't 100 the maxLiquiditySpread?
Shouldn't setting the value to 100 work?

Reply:
if 100 we don't send. recent decision